src/System.Private.CoreLib/shared/System/Threading/Tasks/ValueTask.cs

```diff
@@ -776,6 +776,7 @@ public TResult Result
         public ConfiguredValueTaskAwaitable<TResult> ConfigureAwait(bool continueOnCapturedContext) =>
             new ConfiguredValueTaskAwaitable<TResult>(new ValueTask<TResult>(_obj, _result, _token, continueOnCapturedContext));
 
+#pragma warning disable CS8609 // TODO-NULLABLE: Covariant return types (https://github.com/dotnet/roslyn/issues/23268)
```

Comment:
This shouldn't be needed once ValueType.ToString is fixed.

Reply:
Fixed.